### PR TITLE
Raconteur: add theme

### PR DESCRIPTION
**Pull request by henriqueiamarino**
A theme to promote marvelous stories, Raconteur displays enlarged imagery and typography on a practical layout to attract readers to your narrative.

[Demo site](https://raconteur.mystagingwebsite.com/)

screenshot

### CODE REVIEW

**Comment by scruffian** [comment]
Oh sorry I just pushed over this, hope you didn't lose anything.

**Comment by henriqueiamarino** [comment]
What should I do? Should I add the theme again?

**Comment by henriqueiamarino** [comment]
Hey @scruffian, I am trying to figure out what happens once you force-push something over a PR. I am a newbie on GH. Anyway, as I see patterns from another theme here, I believe it's better if I commit to the theme again. Is that correct?

Screenshot 2024-09-13 at 17 49 44

**Comment by henriqueiamarino** [comment]
I won't be here for a week, so I'll attach the proper theme file to this conversation, @MaggieCabrera. This should be committed to the PR. I tried, but I can't after @scruffian force-pushed another version.

I'll keep an eye on it and reply until Monday morning if someone shows up.
[raconteur.zip](https://github.com/user-attachments/files/16996481/raconteur.zip)

**Comment by MaggieCabrera** [comment]
I updated this branch with Henrique's changes

File: psychedeli/patterns/header-home.php
Comment (alaczek) [inline psychedeli/patterns/header-home.php:1]: Could we please remove all the psychedeli-related code? I think this was included in this PR by mistake.
cc @MaggieCabrera

**Pull request merged**